corehq/apps/userreports/ui/fields.py

```diff
@@ -47,6 +47,9 @@ def prepare_value(self, value):
             return value
 
     def to_python(self, value):
+        if isinstance(value, self.expected_type):
+            return json.loads(json.dumps(value))
```

Comment:
What problem does this solve? Or rather what previous assumption about values being passed in, no longer holds?

Reply:
Do unit tests for this already exist? It would be cool to add a test that this change causes to pass.